### PR TITLE
Set mixer contract config in genesis

### PR DESCRIPTION
**Pull request by alpe**
Follow up on #386 

~~Note to reviewer: tests do fail with my second commit. The sigmoid valued used in tests and config before were not equal.
I need help to adjust them to the expected values~~

Reverted the changes to the test after discussing with @maurolacy

### CODE REVIEW

File: docs/proto/proto-docs.md [hunk 1]
```diff
@@ -329,6 +331,39 @@ ImportDump has all module data for non seed mode.
...
+<a name="confio.poe.v1beta1.MixerContractConfig"></a>
+
+### MixerContractConfig
+Mixer contract setup. See
+https://github.com/confio/poe-contracts/tree/main/contracts/tg4-mixer
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `sigmoid` | [MixerContractConfig.Sigmoid](#confio.poe.v1beta1.MixerContractConfig.Sigmoid) |  | Sigmoid returns a sigmoid-like value of staked amount times engagement points. See the Proof-of-Engagement white-paper for details. |
```
Comment (ethanfrey) [inline docs/proto/proto-docs.md:343]: I would like to support the other variants as well. I can define these on a branch off of this.

File: proto/confio/poe/v1beta1/genesis.proto [hunk 2]
```diff
@@ -87,6 +87,28 @@ message SeedContracts {
...
+
+  MixerContractConfig mixer_contract_config = 14
+      [ (gogoproto.jsontag) = "mixer_contract_config,omitempty" ];
+}
+
+// Mixer contract setup. See
+// https://github.com/confio/poe-contracts/tree/main/contracts/tg4-mixer
+message MixerContractConfig {
+  message Sigmoid {
+    uint64 max_rewards = 1;
```
Comment (alpe) [inline proto/confio/poe/v1beta1/genesis.proto:99]: Note here and others: this is going to be renamed to `max_points` for PoE contracts v0.11.0

File: proto/confio/poe/v1beta1/genesis.proto [hunk 3]
```diff
@@ -87,6 +87,28 @@ message SeedContracts {
...
+    string p = 2 [
+      (gogoproto.nullable) = false,
+      (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"
+    ];
+    string s = 3 [
+      (gogoproto.nullable) = false,
+      (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec"
+    ];
+  }
+  // Sigmoid returns a sigmoid-like value of staked amount times engagement
```
Comment (pinosu) [inline proto/confio/poe/v1beta1/genesis.proto:109]: nice 👍

File: x/poe/types/genesis_test.go [hunk 4]
```diff
@@ -712,3 +724,54 @@ func TestValidateArbiterPoolContractConfig(t *testing.T) {
...
+			}).GetSeedContracts().MixerContractConfig,
+			expErr: true,
+		},
+		"sigmoid p unset": {
+			src: *GenesisStateFixture(func(m *GenesisState) {
+				m.GetSeedContracts().MixerContractConfig.Sigmoid.P = sdk.Dec{}
+			}).GetSeedContracts().MixerContractConfig,
+			expErr: true,
+		},
+		"sigmoid s empty": {
```
Comment (pinosu) [inline x/poe/types/genesis_test.go:754]: good tests!

**Review by pinosu — APPROVED** [review]
lgtm 👍

File: docs/proto/proto-docs.md (hunk 1 above)
Comment (ethanfrey) [reply docs/proto/proto-docs.md:343]: I will do that later, please merge without that. It is not required yet

File: x/poe/types/genesis.go [hunk 5]
```diff
@@ -81,6 +81,13 @@ func DefaultGenesisState() *GenesisState {
 					},
 					DisputeCost: sdk.NewCoin(DefaultBondDenom, sdk.NewInt(1_000_000)),
 				},
+				MixerContractConfig: &MixerContractConfig{
+					Sigmoid: MixerContractConfig_Sigmoid{
+						MaxRewards: 1000,
+						P:          sdk.MustNewDecFromStr("0.62"),
```
Comment (ethanfrey) [inline x/poe/types/genesis.go:87]: I picked new values from my model. Let's try with these and see how simulations work.

**Review by ethanfrey — APPROVED** [review]
Let's get this in. I played with the values a bit. If tests pass, then let's merge these as good basis for others. I can add other equations later.

**Comment by ethanfrey** [comment]
I left a low-priority follow up issue here: https://github.com/confio/tgrade/issues/399
And adjusted the parameters to provide desired results.

This is good to go for mainnet. We can extend later, especially if a new network will use this implementation.

**Pull request merged**